### ISSUE TITLE
[BUG]  'No such container' error message when trying to start new container

### ISSUE THREAD

**Issue by reybango** [ThePorgs/Exegol#147]
### Describe the bug

When trying to create a new container using the 'light' image, I get the following error message:
```
[*] Creating new exegol container                                                                                                                                 
[+] Exegol container successfully created !                                                                                                                       
[*] Location of the exegol workspace on the host : /home/haribo/.exegol/workspaces/demo                                                                           
[+] Opening shell in Exegol 'demo'                                                                                                                                
Error response from daemon: No such container: 5f7ad050b2de7c71d4fabd9213b283930998f223a0d50994df9505b05a4fcb1a
```

![image](https://user-images.githubusercontent.com/1748/224885294-52442a50-e9d2-4d57-8621-192d4aa5eeb5.png)
![image](https://user-images.githubusercontent.com/1748/224885654-e034b7f1-c8a1-4095-9e15-913c4ea56c30.png)
![image](https://user-images.githubusercontent.com/1748/224885710-9c0355a8-67a6-4c31-9b87-f72f236a7b62.png)

### Steps To Reproduce

1. ```exegol start``` and followed the prompts to create the new container from the 'light' image

Error will follow afterwards.

### Exegol Wrapper Version

```Text
[*] Exegol is currently in version v4.1.1
[*] Exegol Discord serv.: https://discord.gg/cXThyp7D6P
[*] Exegol documentation: https://exegol.rtfd.io/
[+] We thank Capgemini for supporting the project (helping with dev) 🙏                                                                                           
[+] We thank HackTheBox for sponsoring the multi-arch support 💚                                                                                                  
[D] Pip installation: On ✔                                                                                                                                        
[D] Git source installation: Off 🪓                                                                                                                               
[D] Host OS: Linux                                                                                                                                                
[D] Arch: amd64                                                                                                                                                   
[D] Raw arch: x86_64                                                                                                                                              
[D] Docker desktop: Off 🪓                                                                                                                                        
[D] Shell type: Linux                                                                                                                                             
[D] Last update check: 13/03/2023
```

### Host OS

Linux

### Configuration of the concerned container

```Text
[*] Exegol is currently in version v4.1.1
[*] Exegol Discord serv.: https://discord.gg/cXThyp7D6P
[*] Exegol documentation: https://exegol.rtfd.io/
[+] We thank Capgemini for supporting the project (helping with dev) 🙏                                                                                           
[+] We thank HackTheBox for sponsoring the multi-arch support 💚                                                                                                  
[V] Listing user configurations                                                                                                                                   

🧠 User configurations                                     
┌─────────────────────────────────────────────────────────┐
│ User config file: /home/haribo/.exegol/config.yml       │
│ Private workspace: /home/haribo/.exegol/workspaces      │
│ Exegol resources: /home/haribo/.exegol/exegol-resources │
│ My resources: /home/haribo/.exegol/my-resources         │
│ Auto-check updates: On ✔                                │
│ Auto-remove images: On ✔                                │
│ Auto-update fs: Off 🪓                                  │
│ Default start shell: zsh                                │
│ Shell logging method: asciinema                         │
│ Shell logging compression: On ✔                         │
└─────────────────────────────────────────────────────────┘


⭐ Container summary                                                                    
┌──────────────────┬───────────────────────────────────────────────────────────────────┐
│             Name │ demo                                                              │
│            Image │ light - v.3.0.1 (Up to date) (amd64)                              │
├──────────────────┼───────────────────────────────────────────────────────────────────┤
│    Creation date │ 13/03/2023 23:18                                                  │
│              GUI │ On ✔                                                              │
│          Network │ host                                                              │
│         Timezone │ On ✔                                                              │
│ Exegol resources │ On ✔ (/opt/resources)                                             │
│     My resources │ On ✔ (/opt/my-resources)                                          │
│    Shell logging │ Off 🪓                                                            │
│       Privileged │ Off ✔                                                             │
│        Workspace │ Dedicated (/workspace)                                            │
│             Envs │ DISPLAY=:0                                                        │
│                  │ _JAVA_AWT_WM_NONREPARENTING=1                                     │
│                  │ QT_X11_NO_MITSHM=1                                                │
│                  │ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin │
│          Volumes │ /etc/timezone ➡ /etc/timezone (RO)                                │
│                  │ /etc/localtime ➡ /etc/localtime (RO)                              │
│                  │ /home/haribo/.exegol/my-resources ➡ /opt/my-resources             │
│                  │ /home/haribo/.exegol/exegol-resources ➡ /opt/resources            │
│                  │ /home/haribo/.exegol/workspaces/demo ➡ /workspace                 │
│                  │ /tmp/.X11-unix ➡ /tmp/.X11-unix                                   │
└──────────────────┴───────────────────────────────────────────────────────────────────┘
```

### Execution logs in debug mode

```Text
exegol start -vvv
[*] Exegol is currently in version v4.1.1
[*] Exegol Discord serv.: https://discord.gg/cXThyp7D6P
[*] Exegol documentation: https://exegol.rtfd.io/
[+] We thank Capgemini for supporting the project (helping with dev) 🙏                                                                                           
[+] We thank HackTheBox for sponsoring the multi-arch support 💚                                                                                                  
[D] Pip installation: On ✔                                                                                                                                        
[D] Git source installation: Off 🪓                                                                                                                               
[D] Host OS: Linux                                                                                                                                                
[D] Arch: amd64                                                                                                                                                   
[D] Raw arch: x86_64                                                                                                                                              
[D] Docker desktop: Off 🪓                                                                                                                                        
[D] Shell type: Linux                                                                                                                                             
[D] Last update check: 13/03/2023                                                                                                                                 

[*] Starting exegol                                                                                                                                               
[D] Attribute not found in parameters: multicontainertag                                                                                                          
[D] == Loading container : exegol-fault                                                                                                                           
[D] Parsing envs : DISPLAY=:0                                                                                                                                     
[D] Adding env DISPLAY=:0                                                                                                                                         
[D] Parsing envs : _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                  
[D] Adding env _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                      
[D] Parsing envs : QT_X11_NO_MITSHM=1                                                                                                                             
[D] Adding env QT_X11_NO_MITSHM=1                                                                                                                                 
[D] Parsing envs : PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                              
[D] Adding env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                                  
[D] Parsing label : org.exegol.app                                                                                                                                
[D] Parsing label : org.exegol.build_date                                                                                                                         
[D] Parsing label : org.exegol.metadata.creation_date                                                                                                             
[D] Parsing label : org.exegol.src_repository                                                                                                                     
[D] Parsing label : org.exegol.tag                                                                                                                                
[D] Parsing label : org.exegol.version                                                                                                                            
[D] Capabilities : []                                                                                                                                             
[D] Load devices : []                                                                                                                                             
[D] Parsing mount : {'Type': 'bind', 'Source': '/tmp/.X11-unix', 'Destination': '/tmp/.X11-unix', 'Mode': '', 'RW': True, 'Propagation': 'rprivate'}              
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/timezone', 'Destination': '/etc/timezone', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}               
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/localtime', 'Destination': '/etc/localtime', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}             
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/my-resources', 'Destination': '/opt/my-resources', 'Mode': '', 'RW': True, 'Propagation':    
'rprivate'}                                                                                                                                                       
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/exegol-resources', 'Destination': '/opt/resources', 'Mode': '', 'RW': True, 'Propagation':   
'rprivate'}                                                                                                                                                       
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/workspaces/fault', 'Destination': '/workspace', 'Mode': '', 'RW': True, 'Propagation':       
'rprivate'}                                                                                                                                                       
[D] Loading workspace volume source : /home/haribo/.exegol/workspaces/fault                                                                                       
[D] Private workspace detected                                                                                                                                    
[D] └── light   → (remote) sha256:7f4cf4a55f34cc2419b57c8b4286be4e5a053316ecce2e23cce5f17a38590571                                                                
[D] == Loading container : exegol-demo                                                                                                                            
[D] Parsing envs : DISPLAY=:0                                                                                                                                     
[D] Adding env DISPLAY=:0                                                                                                                                         
[D] Parsing envs : _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                  
[D] Adding env _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                      
[D] Parsing envs : QT_X11_NO_MITSHM=1                                                                                                                             
[D] Adding env QT_X11_NO_MITSHM=1                                                                                                                                 
[D] Parsing envs : PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                              
[D] Adding env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                                  
[D] Parsing label : org.exegol.app                                                                                                                                
[D] Parsing label : org.exegol.build_date                                                                                                                         
[D] Parsing label : org.exegol.metadata.creation_date                                                                                                             
[D] Parsing label : org.exegol.src_repository                                                                                                                     
[D] Parsing label : org.exegol.tag                                                                                                                                
[D] Parsing label : org.exegol.version                                                                                                                            
[D] Capabilities : []                                                                                                                                             
[D] Load devices : []                                                                                                                                             
[D] Parsing mount : {'Type': 'bind', 'Source': '/tmp/.X11-unix', 'Destination': '/tmp/.X11-unix', 'Mode': '', 'RW': True, 'Propagation': 'rprivate'}              
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/timezone', 'Destination': '/etc/timezone', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}               
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/localtime', 'Destination': '/etc/localtime', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}             
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/my-resources', 'Destination': '/opt/my-resources', 'Mode': '', 'RW': True, 'Propagation':    
'rprivate'}                                                                                                                                                       
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/exegol-resources', 'Destination': '/opt/resources', 'Mode': '', 'RW': True, 'Propagation':   
'rprivate'}                                                                                                                                                       
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/workspaces/demo', 'Destination': '/workspace', 'Mode': '', 'RW': True, 'Propagation':        
'rprivate'}                                                                                                                                                       
[D] Loading workspace volume source : /home/haribo/.exegol/workspaces/demo                                                                                        
[D] Private workspace detected                                                                                                                                    
[D] └── light   → (remote) sha256:7f4cf4a55f34cc2419b57c8b4286be4e5a053316ecce2e23cce5f17a38590571                                                                
[D] == Loading container : exegol-default                                                                                                                         
[D] Parsing envs : DISPLAY=:0                                                                                                                                     
[D] Adding env DISPLAY=:0                                                                                                                                         
[D] Parsing envs : _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                  
[D] Adding env _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                      
[D] Parsing envs : QT_X11_NO_MITSHM=1                                                                                                                             
[D] Adding env QT_X11_NO_MITSHM=1                                                                                                                                 
[D] Parsing envs : PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                              
[D] Adding env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                                  
[D] Parsing label : org.exegol.app                                                                                                                                
[D] Parsing label : org.exegol.build_date                                                                                                                         
[D] Parsing label : org.exegol.metadata.creation_date                                                                                                             
[D] Parsing label : org.exegol.src_repository                                                                                                                     
[D] Parsing label : org.exegol.tag                                                                                                                                
[D] Parsing label : org.exegol.version                                                                                                                            
[D] Capabilities : []                                                                                                                                             
[D] Load devices : []                                                                                                                                             
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/workspaces/default', 'Destination': '/workspace', 'Mode': '', 'RW': True, 'Propagation':     
'rprivate'}                                                                                                                                                       
[D] Loading workspace volume source : /home/haribo/.exegol/workspaces/default                                                                                     
[D] Private workspace detected                                                                                                                                    
[D] Parsing mount : {'Type': 'bind', 'Source': '/tmp/.X11-unix', 'Destination': '/tmp/.X11-unix', 'Mode': '', 'RW': True, 'Propagation': 'rprivate'}              
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/timezone', 'Destination': '/etc/timezone', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}               
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/localtime', 'Destination': '/etc/localtime', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}             
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/my-resources', 'Destination': '/opt/my-resources', 'Mode': '', 'RW': True, 'Propagation':    
'rprivate'}                                                                                                                                                       
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/haribo/.exegol/exegol-resources', 'Destination': '/opt/resources', 'Mode': '', 'RW': True, 'Propagation':   
'rprivate'}                                                                                                                                                       
[D] └── light   → (remote) sha256:7f4cf4a55f34cc2419b57c8b4286be4e5a053316ecce2e23cce5f17a38590571                                                                

👽 Available containers                                                                                                                                           
┌──────────────┬───────────────┬─────────┬───────────────┬─────────────────────────┬─────────────────────────────┬─────────┬───────┬─────────────────────────────┐
│ Id           │ Container tag │ State   │ Image tag     │ Configurations          │ Mounts                      │ Devices │ Ports │ Envs                        │
├──────────────┼───────────────┼─────────┼───────────────┼─────────────────────────┼─────────────────────────────┼─────────┼───────┼─────────────────────────────┤
│ cd64b11a661d │ fault         │ Running │ light (amd64) │ Privileged: Off ✔       │ /tmp/.X11-unix ➡            │         │       │ DISPLAY=:0                  │
│              │               │         │               │ GUI: On ✔               │ /tmp/.X11-unix              │         │       │ _JAVA_AWT_WM_NONREPARENTIN… │
│              │               │         │               │ Network mode: host      │ /etc/timezone ➡             │         │       │ QT_X11_NO_MITSHM=1          │
│              │               │         │               │ Share timezone: On ✔    │ /etc/timezone (RO)          │         │       │ PATH=/usr/local/sbin:/usr/… │
│              │               │         │               │ Exegol resources: On ✔  │ /etc/localtime ➡            │         │       │                             │
│              │               │         │               │ My resources: On ✔      │ /etc/localtime (RO)         │         │       │                             │
│              │               │         │               │ Shell logging: Off 🪓   │ /home/haribo/.exegol/my-re… │         │       │                             │
│              │               │         │               │                         │ ➡ /opt/my-resources         │         │       │                             │
│              │               │         │               │                         │ /home/haribo/.exegol/exego… │         │       │                             │
│              │               │         │               │                         │ ➡ /opt/resources            │         │       │                             │
│              │               │         │               │                         │ /home/haribo/.exegol/works… │         │       │                             │
│              │               │         │               │                         │ ➡ /workspace                │         │       │                             │
│              │               │         │               │                         │                             │         │       │                             │
│ 5f7ad050b2de │ demo          │ Running │ light (amd64) │ Privileged: Off ✔       │ /tmp/.X11-unix ➡            │         │       │ DISPLAY=:0                  │
│              │               │         │               │ GUI: On ✔               │ /tmp/.X11-unix              │         │       │ _JAVA_AWT_WM_NONREPARENTIN… │
│              │               │         │               │ Network mode: host      │ /etc/timezone ➡             │         │       │ QT_X11_NO_MITSHM=1          │
│              │               │         │               │ Share timezone: On ✔    │ /etc/timezone (RO)          │         │       │ PATH=/usr/local/sbin:/usr/… │
│              │               │         │               │ Exegol resources: On ✔  │ /etc/localtime ➡            │         │       │                             │
│              │               │         │               │ My resources: On ✔      │ /etc/localtime (RO)         │         │       │                             │
│              │               │         │               │ Shell logging: Off 🪓   │ /home/haribo/.exegol/my-re… │         │       │                             │
│              │               │         │               │                         │ ➡ /opt/my-resources         │         │       │                             │
│              │               │         │               │                         │ /home/haribo/.exegol/exego… │         │       │                             │
│              │               │         │               │                         │ ➡ /opt/resources            │         │       │                             │
│              │               │         │               │                         │ /home/haribo/.exegol/works… │         │       │                             │
│              │               │         │               │                         │ ➡ /workspace                │         │       │                             │
│              │               │         │               │                         │                             │         │       │                             │
│ 65a6c57c7ad3 │ default       │ Running │ light (amd64) │ Privileged: Off ✔       │ /home/haribo/.exegol/works… │         │       │ DISPLAY=:0                  │
│              │               │         │               │ GUI: On ✔               │ ➡ /workspace                │         │       │ _JAVA_AWT_WM_NONREPARENTIN… │
│              │               │         │               │ Network mode: host      │ /tmp/.X11-unix ➡            │         │       │ QT_X11_NO_MITSHM=1          │
│              │               │         │               │ Share timezone: On ✔    │ /tmp/.X11-unix              │         │       │ PATH=/usr/local/sbin:/usr/… │
│              │               │         │               │ Exegol resources: On ✔  │ /etc/timezone ➡             │         │       │                             │
│              │               │         │               │ My resources: On ✔      │ /etc/timezone (RO)          │         │       │                             │
│              │               │         │               │ Shell logging: Off 🪓   │ /etc/localtime ➡            │         │       │                             │
│              │               │         │               │                         │ /etc/localtime (RO)         │         │       │                             │
│              │               │         │               │                         │ /home/haribo/.exegol/my-re… │         │       │                             │
│              │               │         │               │                         │ ➡ /opt/my-resources         │         │       │                             │
│              │               │         │               │                         │ /home/haribo/.exegol/exego… │         │       │                             │
│              │               │         │               │                         │ ➡ /opt/resources            │         │       │                             │
│              │               │         │               │                         │                             │         │       │                             │
└──────────────┴───────────────┴─────────┴───────────────┴─────────────────────────┴─────────────────────────────┴─────────┴───────┴─────────────────────────────┘

[*] You can use a name that does not already exist to create a new container                                                                                      
[?] Select a container by its name (fault): demo
[*] Location of the exegol workspace on the host : /home/haribo/.exegol/workspaces/demo                                                                           
[+] Opening shell in Exegol 'demo'                                                                                                                                
[D] Adding xhost ACL to local:exegol-demo                                                                                                                         
[D] Opening shell with: docker exec -ti 5f7ad050b2de7c71d4fabd9213b283930998f223a0d50994df9505b05a4fcb1a zsh                                                      
Error response from daemon: No such container: 5f7ad050b2de7c71d4fabd9213b283930998f223a0d50994df9505b05a4fcb1a
```

**Comment by Dramelac** [comment]
Hello,

Sorry for the inconvenience, that is the first time I see this error / use case ...

Can you run the following commands ?
- `docker ps -a`
- `docker inspect exegol-demo`

Thank you for your report and your help!

**Comment by reybango** [comment]
No worries at all. Thanks for helping me out. I spent some time after trying to troubleshoot it so I did remove some of the containers and tried to recreate them but still had the same error. 

```docker ps -a
CONTAINER ID   IMAGE         COMMAND    CREATED        STATUS                    PORTS     NAMES
88ddcea4948e   hello-world   "/hello"   9 hours ago    Exited (0) 9 hours ago              loving_shockley
a3c370910e2a   hello-world   "/hello"   10 hours ago   Exited (0) 10 hours ago             hungry_austin
```

![image](https://user-images.githubusercontent.com/1748/224992591-ae01baf8-e8f9-4a18-8fa6-a203a3890185.png)

**Comment by reybango** [comment]
@Dramelac I reinstalled everything from scratch and now it's working. I can access the containers based off of the images. Not sure what the issue was but it seems reinstalling everything fixed it.

**Comment by Dramelac** [comment]
Did you install docker in rootless mode thé first time ?
I dont really now why your docker was behaving like that, my only idea is that your where in docker rootless mode and your daemon docker had some user container and root container

TLDR: If you were in docker rootless mode, Exegol wrapper doesn't support it (yet)

**Comment by ShutdownRepo** [comment]
I think he was, as per https://github.com/ThePorgs/Exegol/issues/146

**Comment by reybango** [comment]
@Dramelac not sure but @ShutdownRepo said, I may have been. I've since reinstalled everything from scratch and it's working as expected now.
